source/templates.tex

```diff
@@ -2472,11 +2472,10 @@
 \end{codeblock}
 \end{example}
 \begin{note}
-Because the explicit template argument list follows the function template
-name, and because conversion member function templates and constructor
-member function templates are called without using a function name,
-there is no way to provide an explicit template argument list for these
-function templates.
+There is no syntax to form a \grammarterm{template-id}\iref{temp.names}
+by providing an explicit template argument
+list\iref{temp.arg.explicit} for a conversion member function
+template\iref{class.conv.fct}.
```

Comment:
"conversion member function template" is an awkward phrase, as it splits the term "conversion function". The "member" here seems redundant, since every conversion function [template] is already a member function [template]. So how about replacing that with just "conversion function template"?

Reply:
It was there before...